src/test/java/seedu/address/storage/JsonSerializableAddressBookTest.java

```diff
@@ -44,4 +37,7 @@ public void toModelType_duplicatePersons_throwsIllegalValueException() throws Ex
                 dataFromFile::toModelType);
     }
 
+    */
```

Comment:
you need to update the dummy `.json` to make it work with the new json format. I guess it can be done next time